packages/gdk_pixbuf.rb

```diff
@@ -41,6 +41,7 @@ class Gdk_pixbuf < Package
   depends_on 'glibc' # R
   depends_on 'libpng' # R
   depends_on 'zlibpkg' # R
+
   gnome
```

Comment:
Should we add the environment variables in postinstall to gdk_base.rb?

Reply:
They're already getting added to `env.d` though... so as long as a new shell is opened, the variables would be there.

Reply:
Gotcha.